.github/workflows/trivy.yml

```diff
@@ -2,39 +2,45 @@ name: Trivy vulnerability scanner
 on:
   push:
     branches:
-      - master
+      - main
   pull_request:
 jobs:
-  build:
-    name: Build
+  build-and-scan:
+    name: Scan ${{ matrix.image }}
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - target: docker-build-workspace
```

Comment:
Looks like there is no `dependencies` build target for https://github.com/kaito-project/kaito/blob/main/docker/workspace/Dockerfile, that's why it's complaining about `#1 WARN: FromAsCasing: 'as' and 'FROM' keywords' casing do not match (line 2)`